app/src/components/GameRoutes.tsx

```diff
@@ -52,6 +52,11 @@ function CurrentPlayerProvider(props: {
             variables: {
               gameId: data.games[0].id,
               clientUuid: clientUuid()
+            },
+            context: {
+              headers: {
+                "X-Hasura-Role": "anonymous"
+              }
```

Comment:
doesn't actually matter what role you are when joining a game. jwt and role are ignored, it's accessible by everyone.

![image](https://user-images.githubusercontent.com/2534903/81998459-63b58780-9607-11ea-8327-28298082d718.png)